gradio/blocks.py

```diff
@@ -166,6 +168,12 @@ def get_config(self):
             "style": self._style,
         }
 
+    def get_random_value_on_load(self):
```

Comment:
Thoughts on moving this to `IOComponent` since that should be subclassed by any component that has the concept of a value. (Note that `Block` is more general and includes as children classes like `Row` and `Tabs`)